src/select/index.jsx

```diff
@@ -24,7 +24,14 @@ var Select = UIFramework.Component(class extends UIFramework.Control {
     },
   };
   get defaultValue() {
-    return this.props.options[0].value;
+    let defaultValue;
+    try {
+      defaultValue = this.props.options[0].value;
+    } catch (error) {
+      return null;
```

Comment:
Returning a instant value is more preferred here as by doing so, we avoid unnecessary state/variable mutation.